packages/components/constraints/src/helpers.ts

```diff
@@ -1,10 +1,11 @@
-import { customProperties } from '@commercetools-uikit/design-system';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+type TDesignTokenName = keyof typeof designTokens;
 
 // `null` is derived from `getMaxPropEquivalent`
 const getMaxPropTokenValue = (max: number | string | null) => {
   if (!max) return null;
-  // @ts-expect-error
-  return customProperties[`constraint${max}`];
+  return designTokens[`constraint${max}` as TDesignTokenName];
```

Comment:
I needed to cast since it's a dynamic value

Reply:
Nit: if we want to be more "type safe" in these cases (accessing a property dynamically), we can extract this logic into a function with a switch statement.

Similar examples:
* https://github.com/commercetools/ui-kit/blob/e3b726cc7684e6a4da0fdd251b6b0ee8ffda7815/packages/components/spacings/spacings-stack/src/stack.styles.ts#L17-L32
* https://github.com/commercetools/merchant-center-application-kit/blob/a4e7a3452d8a4bd0cf3f7cef65098c8de2ba0740/packages/application-components/src/components/dialogs/internals/dialog.styles.ts#L9-L26

This way we ensure we pick the correct value and we can decide what to do for unknown values.

Not a must, maybe something to think about.